x-pack/plugin/async-search/src/internalClusterTest/java/org/elasticsearch/xpack/search/CrossClusterAsyncSearchIT.java

```diff
@@ -1324,13 +1323,11 @@ public void testCancelViaTasksAPI() throws Exception {
             assertTrue(searchResponseAfterCancellation.isRunning());
             assertFalse(searchResponseAfterCancellation.getSearchResponse().isTimedOut());
             assertThat(searchResponseAfterCancellation.getSearchResponse().getClusters().getTotal(), equalTo(2));
-            assertThat(searchResponseAfterCancellation.getSearchResponse().getFailedShards(), equalTo(0));
```

Comment:
what type of flaky failures are we ignoring? I wonder if this may hide bugs somehow, although I agree that it is not the main purpose of this test to uncover those.

Reply:
The issues are not reproducible in any useful way and there are general problems with cancellation and IT tests in general. The goal of this test was to ensure that cross-cluster searches get cancelled. So either it stays muted until the general cancellation IT test issues are fixed or we relax this test to focus on the key feature. Armin and others have been working on the tests focused on testing cancellation features per se. 

In addition, if there are sporadic shard failures during a cancellation, even if they are caused a bug, they seem low priority to me, since the search was cancelled and the results are useless.